rel/config.exs

```diff
@@ -24,7 +24,7 @@ use Distillery.Releases.Config,
 environment :prod do
   set include_erts: true
   set include_src: false
-  set cookie: :"${NODE_COOKIE}"
+  set cookie: :"${Base.url_encode64(:crypto.strong_rand_bytes(40))}"
```

Comment:
Hm, I think this will end up having the system try to look up a crazy environment variable. Previously the atom was literally `:"${NODE_COOKIE}"` (i.e. a string with a shell env variable lookup). Since we're not looking it up, I think we can put the value right in the cookie using Elixir string interpolation: `:"#{...}"` instead.